src/main/scala/io/getclump/package.scala

```diff
@@ -0,0 +1,15 @@
+package io
+
+package object getclump {
+
+  private[getclump] implicit val executionContext = scala.concurrent.ExecutionContext.global
```

Comment:
Probably good enough for now, how would we make it configurable? Seems the only way would be to have it in every method signature which would suck.

What about something like `implicit val ec = implicitly[ExecutionContext]` and to force the user to have an implicit in scope themselves?